app/default/helpers.py

```diff
@@ -0,0 +1,16 @@
+import requests
+from app.config import FORMS_SERVICE_PUBLIC_HOST
+
+
+def get_return_to_runner_token(form_name: str, formatted_application_payload):
```

Comment:
🏕️  We are now entering the phase of development where the code we are writing isn't self-explanatory. It would be good to have some docstrings here so future generations understand the context this function lives in.

Reply:
IE This function posts to the xgov form runner so that ....

This also makes it easier to refactor - since we are documenting the function of the code rather than the implementation.